sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/aio/_configuration_async.py

```diff
@@ -1,53 +1,59 @@
 # coding=utf-8
 # --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
+# Licensed under the MIT License. See License.txt in the project root for license information.
 # Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
+
+from typing import Any
+
 from azure.core.configuration import Configuration
 from azure.core.pipeline import policies
 
-from .._version import VERSION
-
+VERSION = "unknown"
 
 class TextAnalyticsClientConfiguration(Configuration):
-    """Configuration for TextAnalyticsClient
+    """Configuration for TextAnalyticsClient.
+
     Note that all parameters used to create this instance are saved as instance
     attributes.
 
-    :param credentials: Credentials needed for the client to connect to Azure.
-    :type credentials: :mod:`A msrestazure Credentials
-     object<msrestazure.azure_active_directory>`
-    :param endpoint: Supported Cognitive Services endpoints (protocol and
-     hostname, for example: https://westus.api.cognitive.microsoft.com).
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: azure.core.credentials.TokenCredential
+    :param endpoint: Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com).
     :type endpoint: str
     """
 
-    def __init__(self, credentials, endpoint, **kwargs):
-
-        if credentials is None:
-            raise ValueError("Parameter 'credentials' must not be None.")
+    def __init__(
+        self,
+        credential: "TokenCredential",
```

Comment:
I know this is in the generated code, but shouldn't this type hint be `AsyncTokenCredential`?

Reply:
i'll fix it in autorest then regenerate